Makefile

```diff
@@ -478,6 +478,7 @@ _test: $(REPORTS) _test_setup
 test_parallel:
 	$(MAKE) start_load_balancer START_LOAD_BALANCER=true
 	skipper make $(SKIPPER_PARAMS) _test_parallel
+	scripts/assisted_deployment.sh set_all_vips_dns
```

Comment:
why is that needed if `set_dns` is called as part of `preparing_for_installation`?

Reply:
To ask NM to reload

Reply:
Could we reload inside `_set_dns()` function?

Reply:
No.  It has to be done outside the container

Reply:
Why? Can we make NetworkManager changes be applied from within as we do to the dnsmasq conf? It requires some socket maybe?

Reply:
It is systemd interface.  Not sure we want to deal with it.  I don't think it is in the scope of this PR